### PR TITLE
Update developer.lang

### PR DESCRIPTION
**Pull request by kemarcin**
I have localized a few lines in this document.

### CODE REVIEW

**Comment by flodolo** [comment]
Hi, thanks for your contribution. Have you already tried contacting the Polish team?
https://wiki.mozilla.org/L10n:Teams:pl

CC @splewako @piotrdrag

**Comment by splewako** [comment]
@kemarcin: thank you for your effort. Unfortunately I see errors in every string you translated. Considering this and only part of the file translated, I doubt that working on this PR makes sense.

**Comment by peiying2** [comment]
@splewako, is this the result of kemarcin not familiar with the glossary terms, style?  Could you summarize the findings so kemarcin can improve in future work.

**Comment by splewako** [comment]
> @splewako, is this the result of kemarcin not familiar with the glossary terms, style?

Sure, it is about Polish grammar and not reinventing English words translations mostly (and some consistency maybe).

> I see that you have provided revised translations for each of his suggestions.

I have no idea what are you talking about.

> Could you summarize the findings so kemarcin can improve in future work?

Future work?

> Here are some categories to consider when providing translation feedback: https://developer.mozilla.org/en-US/docs/L10n_Style_Guide.

Categories?
